GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -156,12 +145,23 @@ private string GetShortcutKeyDisplayString(Command cmd)
         public void DisplayDiffTab()
         {
             var revisions = _revisionGrid.GetSelectedRevisions();
+            DiffFiles.StoreNextIndexToSelect();
+            SetDiffs(revisions);
+            if (DiffFiles.SelectedItem == null)
+            {
+                DiffFiles.SelectStoredNextIndex();
+            }
+        }
+
+        private void SetDiffs(IReadOnlyList<GitRevision> revisions)
```

Comment:
Can be inlined again.

Reply:
As explained in the change above, no inlining.